src/screens/SendCoinsConfirmScreen.tsx

```diff
@@ -21,7 +21,7 @@ const i18n = require('../../loc');
 const ScreenFooter = (onSendPress: () => void, onDetailsPress: () => void, buttonTitle?: string) => (
   <View style={styles.footer}>
     <Button
-      testID="send-coins-confirmation-button"
+      testID="transaction-confirmation-button"
```

Comment:
I prefer "send coins" because the word "transaction" may mean both negative and positive one and in this case, we do have in mind specifically sending coins

Reply:
I changed that to a more generic name because it is used not only in the send section but also in the cancel section.